src/org/labkey/test/tests/InvalidateSessionTest.java

```diff
@@ -0,0 +1,151 @@
+package org.labkey.test.tests;
+
+import org.apache.hc.core5.http.HttpStatus;
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.query.SelectRowsCommand;
+import org.labkey.remoteapi.query.SelectRowsResponse;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.PasswordUtil;
+import org.labkey.test.util.PermissionsHelper;
+import org.openqa.selenium.Cookie;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.fail;
+import static org.labkey.test.WebTestHelper.getCookies;
+
+@Category({Daily.class})
+public class InvalidateSessionTest extends BaseWebDriverTest
+{
+    private static final String USER = "pwd_change_user@invalidatesessiontest.test";
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        InvalidateSessionTest init = getCurrentTest();
+        init.doSetup();
+    }
+
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return null;
+    }
+
+    @Override
+    protected void doCleanup(boolean afterTest)
+    {
+        _userHelper.deleteUsers(afterTest, USER);
+    }
+
+    private void doSetup()
+    {
+        _userHelper.createUser(USER);
+
+        ApiPermissionsHelper permissionsHelper = new ApiPermissionsHelper(this);
+        permissionsHelper.addMemberToRole(USER, "Folder Administrator", PermissionsHelper.MemberType.user);
+        setInitialPassword(USER);
+    }
+
+    /*
+        Regression coverage for Secure Issue 51523: Invalidate sessions on password change
+     */
+    @Test
+    public void testSessionInvalidatesAfterPasswordChange() throws IOException
+    {
+        signOut();
+        signIn(USER);
+        Connection cn = createDefaultConnection();
+        SelectRowsResponse response;
+        SelectRowsCommand selectCmd = new SelectRowsCommand("auditLog", "UserAuditEvent");
+        try
+        {
+            response = selectCmd.execute(cn, "Home");
+            Assert.assertEquals("Did not establish the database connection before the password change", 200,
+                    response.getStatusCode());
+        }
+        catch (IOException | CommandException e)
+        {
+            throw new RuntimeException(e);
+        }
```

Comment:
No need for a try/catch here. The method can just declare that it throws these exceptions. It already has `throws IOException`